Segment_Tree_With_Lazi_Propagation/Segment_Tree_With_Lazy_Propagation.py

```diff
@@ -1,12 +1,16 @@
-'''Segment Tree with Lazy Propagation is used when you want to perform some operation within an array range with range updation. Let's us suppose you want to perform minimum operation in certain range[a:b+1] and after performing minimum operation you are updating array index in range [l:r+1]. If you apply brute force approach then you will end up with O(n^2) complexity. If you use segment tree then your algorithm complexity will be O(nlogn) i.e. more efficient than bruteforce approach.
+'''Segment Tree with Lazy Propagation is used when you want to perform some operation within an array range with 
```

Comment:
Try to limit the number of characters in each line to 80 characters (including indentation)

Reply:
Done